recipe/meta.yaml

```diff
@@ -77,7 +77,9 @@ requirements:
 test:
   requires:
     - {{ compiler('c') }}
-    - scipy
+    - scipy         # [not (osx and py==38)]
+    # Try to force a scipy version that is currently not available on defaults.
+    - scipy >=1.3.3  # [osx and py==38]
```

Comment:
The previous build with `openmp 3.*` failed with https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=109431&view=logs&j=aa28db25-cf14-560f-7cf5-b2cf9b1d5f34&t=58ac3b9e-817b-51ec-88c2-b6a556e49d86&l=7461:
```
import: 'neuron.rxd'
Traceback (most recent call last):
  File "/usr/local/miniconda/conda-bld/neuron_1579274288471/test_tmp/run_test.py", line 5, in <module>
    import neuron.rxd
  File "/usr/local/miniconda/conda-bld/neuron_1579274288471/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.8/site-packages/neuron/rxd/__init__.py", line 13, in <module>
    from . import rxd
  File "/usr/local/miniconda/conda-bld/neuron_1579274288471/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.8/site-packages/neuron/rxd/rxd.py", line 8, in <module>
    import scipy.sparse.linalg
  File "/usr/local/miniconda/conda-bld/neuron_1579274288471/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.8/site-packages/scipy/sparse/linalg/__init__.py", line 113, in <module>
    from .isolve import *
  File "/usr/local/miniconda/conda-bld/neuron_1579274288471/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.8/site-packages/scipy/sparse/linalg/isolve/__init__.py", line 6, in <module>
    from .iterative import *
  File "/usr/local/miniconda/conda-bld/neuron_1579274288471/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.8/site-packages/scipy/sparse/linalg/isolve/iterative.py", line 10, in <module>
    from . import _iterative
ImportError: dlopen(/usr/local/miniconda/conda-bld/neuron_1579274288471/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.8/site-packages/scipy/sparse/linalg/isolve/_iterative.cpython-38-darwin.so, 2): Library not loaded: @rpath/libopenblas.dylib
  Referenced from: /usr/local/miniconda/conda-bld/neuron_1579274288471/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.8/site-packages/scipy/sparse/linalg/isolve/_iterative.cpython-38-darwin.so
  Reason: image not found
```
For some reason it got

```
scipy:              1.3.1-py38h1a1e112_0             defaults
```
Let's see if the solver outputs some error if we force it to get `conda-forge::scipy`...

Reply:
The build after that failed with
```
Package libgfortran-ng conflicts for:
numpy -> libgfortran-ng[version='>=3.0']
scipy[version='>=1.3.3'] -> libblas[version='>=3.8.0,<4.0a0'] -> openblas[version='>=0.3.6,<0.3.7.0a0'] -> libgfortran-ng[version='>=3.0']
Package libgfortran conflicts for:
openmpi[version='>=3.1.4,<3.2.0a0'] -> libgfortran[version='>=3.0.1,<4.0.0.a0']
scipy[version='>=1.3.3'] -> libgfortran[version='>=4.0.0,<5.0.0.a0']
numpy -> blas==1.1=openblas -> libgfortran[version='>=4.0.0,<5.0.0.a0']
```
which would mean `openmpi` isn't build for `libgfortran>=4`, I guess.

I'm giving an update+patch a try now...